src/main/java/com/bandwidth/iris/sdk/model/DisconnectTelephoneNumberOrderResponse.java

```diff
@@ -7,6 +7,11 @@
 @XmlRootElement(name = "DisconnectTelephoneNumberOrderResponse")
 @XmlAccessorType(XmlAccessType.FIELD)
 public class DisconnectTelephoneNumberOrderResponse extends BaseResponse {
+
+    @XmlElementWrapper(name = "DisconnectedTelephoneNumberList")
+    @XmlElement(name = "TelephoneNumber")
+    private List<String> disconnecteedNumbers = new ArrayList<String>();
```

Comment:
Hey,

Is this the same number list that is linked in the `DisconnectTelephoneNumberOrderResponse.getorderRequest().getDisconnectTelephoneNumberOrderType().getTelephoneNumberList()`  of the current SDK?

Reply:
It is not, or at least they map to different XML elements obviously.
So I am assuming this
1. DisconnectTelephoneNumberOrderResponse.getorderRequest().getDisconnectTelephoneNumberOrderType().getTelephoneNumberList() comes from the numbers included in the disconnect order
2. The DisconnectedTelephoneNumberList XML element in the disconnect order status response was not being mapped to anything, so that's what I am mapping new here. From my testing, this list includes only those numbers that were successfully disconnected, which is the key piece of information that was missing prior to this change.

Reply:
Can you provide an example of the XML response that has the 
`<DisconnectedTelephoneNumberList>` element?

Reply:
Ha, we crossed each other. I'd edited my earlier comment to provide an example of that, which you should see below

Reply:
Also, if you look at the unit test I added, you will see the same content in ``` IrisClientTestUtils.validCreateDisconnectTelephoneNumberOrderResponseSuccess ```

Reply:
Got it!  That is the information I needed thanks!

Reply:
I also now see the examples provided for the test cases 👍 thank you for those.